cmd/table-manager/main.go

```diff
@@ -20,12 +20,18 @@ func main() {
 				middleware.ServerUserHeaderInterceptor,
 			},
 		}
+		awsClientConfig    = chunk.AWSStorageConfig{}
 		tableManagerConfig = chunk.TableManagerConfig{}
 	)
-	util.RegisterFlags(&serverConfig, &tableManagerConfig)
+	util.RegisterFlags(&serverConfig, &awsClientConfig, &tableManagerConfig)
 	flag.Parse()
 
-	tableManager, err := chunk.NewDynamoTableManager(tableManagerConfig)
+	awsClient, tableName, err := chunk.NewAWSStorageClient(awsClientConfig)
```

Comment:
Again, it should not construct storageClient IMO, soon there will be someone that will want to use PutObject on this which would end up with nil panic.